diff -Naur oiio-2.2.15.1_copy/src/cmake/modules/FindOpenEXR.cmake oiio-2.2.15.1/src/cmake/modules/FindOpenEXR.cmake
--- oiio-2.2.15.1_copy/src/cmake/modules/FindOpenEXR.cmake	2021-06-04 05:50:22.000000000 +0800
+++ oiio-2.2.15.1/src/cmake/modules/FindOpenEXR.cmake	2022-02-21 22:43:21.327777081 +0800
@@ -34,12 +34,7 @@
 #
 #
 
-# First, try to fine just the right config files
-find_package(Imath CONFIG)
-if (NOT TARGET Imath::Imath)
-    # Couldn't find Imath::Imath, maybe it's older and has IlmBase?
-    find_package(IlmBase CONFIG)
-endif ()
+find_package(IlmBase CONFIG)
 find_package(OpenEXR CONFIG)
 
 if (TARGET OpenEXR::OpenEXR AND TARGET Imath::Imath)
